package.json

```diff
@@ -41,6 +41,7 @@
 		"@typescript-eslint/eslint-plugin": "^5.48.2",
 		"@typescript-eslint/parser": "^5.48.2",
 		"@vitest/coverage-istanbul": "^0.29.0",
+		"all-contributors-cli": "^6.24.0",
```

Comment:
Ahh, one more thing here, [Knip]() doesn't know that this is used:

```plaintext
Run pnpm lint:knip

> template-typescript-node-package@1.2[7](https://github.com/JoshuaKGoldberg/template-typescript-node-package/actions/runs/4484576327/jobs/7885225024?pr=315#step:4:8).5[8](https://github.com/JoshuaKGoldberg/template-typescript-node-package/actions/runs/4484576327/jobs/7885225024?pr=315#step:4:9) lint:knip /home/runner/work/template-typescript-node-package/template-typescript-node-package
> knip

Unused dev dependencies (1)
all-contributors-cli  package.json
 ELIFECYCLE  Command failed with exit code 1.
```

The root issue is that Knip doesn't recognize [zx](https://github.com/google/zx) scripts. I'll try to see if I can contribute that!

Reply:
https://github.com/webpro/knip/issues/96